pkg/curatedpackages/packagecontrollerclient.go

```diff
@@ -235,28 +269,6 @@ func (pc *PackageControllerClient) IsInstalled(ctx context.Context) bool {
 	return bool && err == nil
 }
 
-func (pc *PackageControllerClient) ApplySecret(ctx context.Context) error {
```

Comment:
Does this mean we plan on overriding the helm values rather than applying the secret to the cluster?

Reply:
Right, now the secrets are not created by CLI but become part of the helm chart.